src/client/util/http.cc

```diff
@@ -135,6 +139,10 @@ bool HTTP::DownloadViaClearnet() {
           | SSL_OP_NO_TLSv1 | SSL_OP_NO_TLSv1_1
           | SSL_OP_NO_SSLv2 | SSL_OP_NO_SSLv3);
   }
+  else
+    LOG(debug) << "HTTP: Skipped Reseed SSL checking ";
```

Comment:
FYI: because the if block has braces, else should also have braces; even if one line (google style thing).